### PR TITLE
Tuist 0.13.0 release

### PR DESCRIPTION
**Pull request by kwridan**
- Adding a post to highlight some of the new features of Tuist 0.13.0
- Introduced new `.posts__post-screenshot` css class to help style screenshots

### CODE REVIEW

**Comment by kwridan** [comment]
Preview: https://deploy-preview-30--peaceful-fermat-c0d5d7.netlify.com/blog/2019/04/01/version-0.13.0/

Screenshot 2019-04-01 at 20 28 42

too much shadows perhaps?

Without styling it looked a little bland:

Screenshot 2019-04-01 at 20 42 58

File: collections/_posts/2019-04-01-version-0.13.0.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+     alt="Unit tests target with host application" 
+     src="/images/0.13.0/0.13.0-unit-tests-host-application.png" />
+
+<img class="posts__post-screenshot" 
+     alt="UI tests target with target application" 
+     src="/images/0.13.0/0.13.0-ui-tests-target-application.png" />
+
+## What’s next?
+
+- [#265](https://github.com/tuist/tuist/issues/265) Storyboard support - to generate via (`tuist init`) as well as specify in the `Project.swift` manifest to include in the generated Xcode project.
```
Comment (ollieatkinson) [inline collections/_posts/2019-04-01-version-0.13.0.md]: I'd reword this one slightly, because tuist already has Storyboard support. You mention it in the second sentence but I'd make it more obvious up front.

**Comment by ollieatkinson** [comment]
@kwridan I like the shadows, nice touch.
